src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/UserServiceTest.kt

```diff
@@ -451,8 +451,6 @@ class UserServiceTest {
       assertUserUpdated(user, deliusUser, newProbationRegion)
     }
 
-    // probation area code
-
```

Comment:
I think you can squash the 2 commits to lose this tidying commit?

Reply:
oops!